Add arXiv target to Makefile

matthewfeickert opened the pull request:
The arXiv target prepares a `tar.gz` file with all the necessary files for arXiv to build the document correctly.

matthewfeickert committed:
Add arXiv target to Makefile

The arXiv target prepares a tar.gz file with all the necessary files for
arXiv to build the document correctly.

matthewfeickert committed:
Remove hyperref package from preamble for arXiv

If hyperref is loaded then there is an option clash.
c.f.
https://tex.stackexchange.com/questions/329461/arxiv-option-clash-for-package-hyperref-when-there-are-no-options

omazapa commented:
Hi @matthewfeickert 

When you compile the latex code, in the directory is generated a file main.bbl 
that is required to arxiv, is just to add this to the compressed file.

matthewfeickert commented:
@omazapa Yeah, I guess we just had perfect timing and figured this out about the same time. I'm commiting a Makefile (commit b9607f7c281d591ea18bd136b854a7cb4dce84bd) that moves this in as `ms.bbl`. :+1:

matthewfeickert committed:
Add compile step to generate .bbl file

arXiv needs a .bbl to generate the bibliography, so compile to generate
it and then move that into the directory to tar as ms.bbl

omazapa commented:
HI @matthewfeickert 
ms.tex is the main.tex?
because it works for me with main.tex and main.bbl
Thank you!

matthewfeickert commented:
@omazapa [arXiv gives priority to `ms.*`](https://arxiv.org/help/faq/mistakes#wrongtex), so this is just making things easier by explicitly renaming them in the tar.gz building step.

matthewfeickert committed:
Set color of internal links to refs to be blue
